parsing/g_prim.mlg

```diff
@@ -27,6 +27,13 @@ let my_int_of_string ?loc s =
   with Failure _ ->
     CErrors.user_err ?loc (Pp.str "This number is too large.")
 
+let my_to_nat_string ?loc ispos s =
+  match NumTok.Unsigned.to_nat s with
+  | Some n -> n
+  | None ->
+    let pos = if ispos then "natural" else "integer" in
+    CErrors.user_err ?loc Pp.(str "This number is not a " ++ str pos ++ str " number.")
```

Comment:
I would rather write "an integer", but my english might be wrong.

Reply:
My English is also poor, but I agree with you.